README.md

```diff
@@ -1,289 +1,83 @@
-
- * When you're ready to use LitElement in a project, install it via [npm](https://www.npmjs.com/). To run the project in the browser, a module-compatible toolchain is required. We recommend installing the [Polymer CLI](https://github.com/Polymer/polymer-cli) and using its development server as follows.
-
-    1. Add LitElement to your project:
-
-        ```npm i lit-element```
-
-    1. Install the webcomponents polyfill. If you're developing a reusable package, this should be a dev dependency which you load in your tests, demos, etc.
-
-        ```npm i -D @webcomponents/webcomponentsjs```
-
-
-    1. Create an element by extending LitElement and calling `customElements.define` with your class (see the examples below).
-
-    1. Install the Polymer CLI:
-
-        ```npm i -g polymer-cli```
-
-    1. Run the development server and open a browser pointing to its URL:
-
-        ```polymer serve```
-
-    > LitElement is published on [npm](https://www.npmjs.com/package/lit-element) using JavaScript Modules.
-    This means it can take advantage of the standard native JavaScript module loader available in all current major browsers.
-    >
-    > However, since LitElement uses npm convention to reference dependencies by name, a light transform to rewrite specifiers to URLs is required to get it to run in the browser. The polymer-cli's development server `polymer serve` automatically handles this transform.
-
-    Tools like [WebPack](https://webpack.js.org/) and [Rollup](https://rollupjs.org/) can also be used to serve and/or bundle LitElement.
-
-
-## Minimal Example
-
-  1. Create a class that extends `LitElement`.
-  1. Use a `@property` decorator to create a property (or implement a static `properties`
-  getter that returns the element's properties). (which automatically become observed attributes).
-  1. Then implement a `render()` method and use the element's
-current properties to return a `lit-html` template result to render
-into the element.
-
-```html
-  <script src="node_modules/@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
-  <script type="module">
-    import {LitElement, html} from 'lit-element';
+```ts
+    import {LitElement, html, css} from 'lit-element';
 
-    class MyElement extends LitElement {
+    // This decorator defines the element.
+    @customElement('my-element');
+    export class MyElement extends LitElement {
 
-      static get properties() {
-        return {
-          mood: {type: String}
-        };
-      }
+      // This decorator creates a property accessor that triggers rendering and
+      // an observed attribute.
+      @property()
+      mood = 'great';
 
-      constructor() {
-        super();
-        this.mood = 'happy';
-      }
+      // Provide styling for the element.
+      static styles = css`.mood { color: green; }`;
```

Comment:
Why not use an id as a selector to show of that that works in shadow dom? also class and property both named "mood" is maybe confusing.

Reply:
Simplified